### DISCUSSION TITLE
Feature Request: Option to only allow the assignees on a PR to merge the PR

### DISCUSSION THREAD

**Discussion by IDrumsey** [community/community#86605] [Pull Requests]
### Select Topic Area

General

### Body

Kinda going off this [so](https://stackoverflow.com/questions/41087206/on-github-whats-the-difference-between-reviewer-and-assignee), the "assignee" has a loose definition in a PR. In my mind, I see the assignee as the one ultimately responsible for getting the PR merged, so in the past I've had issues with reviewers merging instead of the assignee since there's no enforcement of that standard. Maybe there could be a feature that allows repo admins to enforce only allowing PR assignees to merge PRs. Just a thought.

**Comment by queenofcorgis** [comment]
Hey there - thanks for sharing this feedback - I've added the appropriate label so it can be reviewed by our team. If you or anyone else has any use cases, perspectives, or requests related to this feedback, please continue to share it here!

**Reply by IDrumsey** [reply]
Awesome, thanks!

**Reply by ATVIgathompson** [reply]
Possibly reviving this topic. Was there any traction whatsoever on this? This is something that our teams would really benefit from as well. We'd like our developers to have the agency and responsibility to merge their own PRs while still requiring things like a minimum number of reviews and review by code owners.
